plutarch-docs/src/Typeclasses/PLiftable.md

```diff
@@ -0,0 +1,231 @@
+# `PLiftable`
```

Comment:
We can technically make this into a literate haskell where we can have snippet that will be compiled to check validity. But this document looks good without it. We can probably reserve that for other guides more oriented for usages.